index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css" type="text/css" media="all" />
+    <title>Capstone Website</title>
+</head>
+<body class="body">
+  <section class="content">
+    <header id="top-bar">
+      <div class="so-container">
+        <div class="socials">
+            <img id="facebook" src="./images/socials/facebook.svg" alt="Facebook Handle.">
+            <img id="twitter" src="./images/socials/twitter.svg" alt="Twitter Handle.">
+            <ul class="nav-list">
+                <li id="top-1">English</li>
+                <li id="top-2">My Page</li>
+                <li id="top-3">Logout</li>
+            </ul>
+            </div>
+            </div>
+            <nav id="nav-bar">
+              <img id="ham-menu" src="./images/ham.png" alt="Hamburger Menu" />
+               <img id="logo" src="./images/nba-logos/nba-logo.svg" alt="Logo">
+               <img id="exit" src="./images/close-button.png" alt="Exit Menu" />
+               <ul id="my-menu">
+                <li id="menu-1"><a id="about" href="about.html">About</a></li>
+               	<li id="menu-2">Program</li>
+                <li id="menu-3">Join</li>
+                <li id="menu-4">Sponsor</li>
+                <li id="menu-5">News</li>
+                <li id="menu-6">Others</li>
+               </ul>
+            </nav>
```

Comment:
- [ ] Great job so far. **Your navigation bar has a different background color and font color from what is in the design to follow.** 
Your design
![image](https://user-images.githubusercontent.com/90222110/183116844-5a7514e2-44f3-4f50-8cb5-8cd73fa988fd.png)

Design given

![image](https://user-images.githubusercontent.com/90222110/183116981-223de3bc-975e-4bf5-b2d1-122a5fc7ef34.png)

Please try to correct your design referring to the design given by changing the above-mentioned colors. Also, you may consider adding some box shadow 😉